src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2950,6 +2952,16 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
                 namespace='Microsoft.Network', type='virtualNetwork',
                 name=vnet_peer
             )
+    if workspace_resource_id is not None:
```

Comment:
In AKS, we create default Azure Log Analytics workspace if the customer passed on the "monitoring" addon and with no workspace-resource-id.  Looks like in ARO case, there is no monitor addon concept and  enabling monitoring governed by --workspace-resource-id parameter.  is that, right? or we will be going with the same behavior as AKS i.e. creating default log analytics workspace when customer provided no workspace-resource-id parameter.

Reply:
ARO doesn't have a concept of add-ons at the moment.  I do not believe we want to create a default log analytics workspace if they do not pass in the `workspace-resource-id`.  We will not enable it if it's not passed in.

Reply:
@troy0820, ok, thanks.